### DISCUSSION TITLE
Welcome to PGSuper Discussions!

### DISCUSSION THREAD

**Discussion by RickBrice** [WSDOT/PGSuper#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dipichandra** [comment]
Exploring the features of PGSuper.

-Dipi Chandra
Assistant Professor
Dept. of Civil & Environmental Engg.
Kennesaw State University
Marietta, GA 30060

**Reply by RickBrice** [reply]
Welcome to PGSuper. If you have any questions you can post here or on our forums at PGSuper.com.
